docs/content/preview/explore/change-data-capture/debezium-connector-yugabytedb.md

```diff
@@ -949,6 +969,16 @@ The APIs used to fetch the changes are set up to work with TLSv1.2 only. Make su
 
 {{< /note >}}
 
+{{< note title="Note" >}}
+
+If you have a YugabyteDB cluster with SSL enabled, all you need to do is obtain the root certificate and provide the path of the file to the `database.sslrootcert` configuration property. You can follow these links to get the certificates to your universe:
+
+* [Local deployments](../../secure/tls-encryption/_index)
+* [YB Anywhere](../../yugabyte-platform/security/enable-encryption-in-transit#connect-to-a-ysql-endpoint-with-tls)
+* [YB Managed](../../yugabyte-cloud/cloud-secure-clusters/cloud-authentication#download-your-cluster-certificate)
```

Comment:
Links should always have a trailing slash, even if you're following the final path element with an anchor link. _index is the default file, so you just omit that. And here are the corrected relative paths :) 
```suggestion
* [Local deployments](../../../secure/tls-encryption/)
* [YB Anywhere](../../../yugabyte-platform/security/enable-encryption-in-transit/#connect-to-a-ysql-endpoint-with-tls)
* [YB Managed](../../../yugabyte-cloud/cloud-secure-clusters/cloud-authentication/#download-your-cluster-certificate)
```